io_ogre/config.py

```diff
@@ -43,6 +43,7 @@
 
     # Materials
     'MATERIALS' : True,
+    'MATERIALS_V2JSON': True,
```

Comment:
```suggestion
    'MATERIALS_V2JSON': False,
```
default behavior should not change

Reply:
However, we already know whether we export for v1 or v2, based on `detect_converter_type()` - so probably this should not be exposed in the UI at all